src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -394,7 +393,9 @@ void staticContrib(const Opm::Group&        group,
                 sGrp[Isi::gasVoidageLimit] = inj_cntl.target_void_fraction;
             }
         }
-        if (phs == Opm::Phase::WATER) {
+
+        if (group.hasInjectionControl(Opm::Phase::WATER)) {
+            const auto& inj_cntl = group.injectionControls(Opm::Phase::GAS, sumState);
             if (inj_cntl.surface_max_rate > 0.) {
```

Comment:
copy-paste bug. Should be Opm::Phase::WATER

Reply:
Thanks - fixed

Reply:
Your fix is gone! This explains the output issue.

Reply:
Dang; fixed now. If there was a price for "most annoying PR" this would certainly be a contender!